dom0/sd-dom0-systemd.sls

```diff
@@ -21,3 +25,14 @@ apply-systemd-changes:
   cmd.run:
     - name: sudo systemctl restart systemd-logind
 {% endif %}
+
+enable-user-units:
+  cmd.run:
+    - names:
+        - systemctl --user daemon-reload
```

Comment:
I wonder if cmd.run will reliably work here with these two commands back to back, or if we need to have the enable command wait for the service to reload. (ie `module.wait` for systemd to restart and then have the timer enabling `watch` the previous state). Will look more into this briefly tm.

Reply:
@rocodes, are you questioning whether `systemctl daemon-reload` is blocking?  [It is.](https://github.com/systemd/systemd/blob/c2e6ed612a11d363dfca71c9a78ea005bfc0cc3a/src/systemctl/systemctl-daemon-reload.c#L40-L41)  But [it looks like `cmd.run(names=[])` is not executed in deterministic order](https://github.com/saltstack/salt/issues/16879#issuecomment-60663390), even though `names` is a list.  This isn't the only state declaration in which we implicitly deterministic ordering of `names`.  Let me tack on a commit to fix that.

Reply:
Sorry, my question was about whether cmd.run would reliably chain commands like that or not, and it looks like you found the answer. Looking at the changes now :)

Reply:
Nice one. It's a bit hard to find documentation from Salt about this syntax but there are a few [examples](https://github.com/saltstack/salt-ci-images/blob/327950581ba196f2cd10e992deab7b9c43133f80/golden-state-tree/pkgs/vault.sls#L14-L18), so lgtm